src/platform/graphics/constants.js

```diff
@@ -970,24 +971,24 @@ export const pixelFormatInfo = new Map([
     [PIXELFORMAT_DEPTH,         { name: 'DEPTH', size: 4 }],
     [PIXELFORMAT_DEPTHSTENCIL,  { name: 'DEPTHSTENCIL', size: 4 }],
     [PIXELFORMAT_111110F,       { name: '111110F', size: 4 }],
-    [PIXELFORMAT_SRGB,          { name: 'SRGB', size: 4 }],
-    [PIXELFORMAT_SRGBA,         { name: 'SRGBA', size: 4 }],
+    [PIXELFORMAT_SRGB8,         { name: 'SRGB8', size: 4 }],
+    [PIXELFORMAT_SRGBA8,        { name: 'SRGBA8', size: 4 }],
     [PIXELFORMAT_BGRA8,         { name: 'BGRA8', size: 4 }],
 
     // compressed formats
-    [PIXELFORMAT_DXT1, { name: 'DXT1', blockSize: 8 }],
-    [PIXELFORMAT_DXT3, { name: 'DXT3', blockSize: 16 }],
-    [PIXELFORMAT_DXT5, { name: 'DXT5', blockSize: 16 }],
-    [PIXELFORMAT_ETC1, { name: 'ETC1', blockSize: 8 }],
-    [PIXELFORMAT_ETC2_RGB, { name: 'ETC2_RGB', blockSize: 8 }],
-    [PIXELFORMAT_ETC2_RGBA, { name: 'ETC2_RGBA', blockSize: 16 }],
-    [PIXELFORMAT_PVRTC_2BPP_RGB_1, { name: 'PVRTC_2BPP_RGB_1', blockSize: 8 }],
-    [PIXELFORMAT_PVRTC_2BPP_RGBA_1, { name: 'PVRTC_2BPP_RGBA_1', blockSize: 8 }],
-    [PIXELFORMAT_PVRTC_4BPP_RGB_1, { name: 'PVRTC_4BPP_RGB_1', blockSize: 8 }],
-    [PIXELFORMAT_PVRTC_4BPP_RGBA_1, { name: 'PVRTC_4BPP_RGBA_1', blockSize: 8 }],
-    [PIXELFORMAT_ASTC_4x4, { name: 'ASTC_4x4', blockSize: 16 }],
-    [PIXELFORMAT_ATC_RGB, { name: 'ATC_RGB', blockSize: 8 }],
-    [PIXELFORMAT_ATC_RGBA, { name: 'ATC_RGBA', blockSize: 16 }],
+    [PIXELFORMAT_DXT1,              { name: 'DXT1', blockSize: 8, srgb: true }],
+    [PIXELFORMAT_DXT3,              { name: 'DXT3', blockSize: 16, srgb: true }],
+    [PIXELFORMAT_DXT5,              { name: 'DXT5', blockSize: 16, srgb: true }],
+    [PIXELFORMAT_ETC1,              { name: 'ETC1', blockSize: 8, srgb: true }],
+    [PIXELFORMAT_ETC2_RGB,          { name: 'ETC2_RGB', blockSize: 8, srgb: true }],
+    [PIXELFORMAT_ETC2_RGBA,         { name: 'ETC2_RGBA', blockSize: 16, srgb: true }],
+    [PIXELFORMAT_PVRTC_2BPP_RGB_1,  { name: 'PVRTC_2BPP_RGB_1', blockSize: 8, srgb: true }],
+    [PIXELFORMAT_PVRTC_2BPP_RGBA_1, { name: 'PVRTC_2BPP_RGBA_1', blockSize: 8, srgb: true }],
+    [PIXELFORMAT_PVRTC_4BPP_RGB_1,  { name: 'PVRTC_4BPP_RGB_1', blockSize: 8, srgb: true }],
+    [PIXELFORMAT_PVRTC_4BPP_RGBA_1, { name: 'PVRTC_4BPP_RGBA_1', blockSize: 8, srgb: true }],
+    [PIXELFORMAT_ASTC_4x4,          { name: 'ASTC_4x4', blockSize: 16, srgb: true }],
+    [PIXELFORMAT_ATC_RGB,           { name: 'ATC_RGB', blockSize: 8, srgb: true }],
+    [PIXELFORMAT_ATC_RGBA,          { name: 'ATC_RGBA', blockSize: 16, srgb: true }],
```

Comment:
Wait, they're all SRGB all the time? Can't I have a linear normal map, say, using these compressed formats?

Reply:
these we convert to linear manually.
Soon I'll add compressed texture formats stored in sRGB as well, separate PR.